backend/languages/java.js

```diff
@@ -0,0 +1,35 @@
+const util = require("util");
+const exec = util.promisify(require("child_process").exec);
+var fs = require("fs");
+
+const java = (input, res, name) => {
+  fs.writeFile(`${name}.txt`, input, (err) => {
+    if (err) res.json({ error: err });
+    exec("docker run -d -it java:v1 /bin/bash").then((resp) => {
```

Comment:
I think it would be better for code clarity if we write the docker run commands on a separate line as variables

Reply:
I'll open a new PR to address these changes